index.html

```diff
@@ -1111,8 +1111,8 @@ <h4>parseDerivedProofValue</h4>
 single <em>derived proof value</em> object is produced as output, which
 contains a set of six or seven elements, having the names "bbsProof",
 "labelMap", "mandatoryIndexes", "selectiveIndexes", "presentationHeader",
-"featureOption", and possibly "pseudonym" depending on the value of the
-|featureOption| parameter.
+"featureOption", and possibly "pseudonym" and/or "lengthBBSMessages" depending
+on the value of the |featureOption| parameter.
```

Comment:
As above — what `value of the |featureOption| parameter` means there will be `a "pseudonym"`, `a "lengthBBSMessages"`, or both.
```suggestion
"featureOption", and, depending on the value of the |featureOption| parameter, 
"pseudonym" and/or "lengthBBSMessages".
```